extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -491,42 +492,42 @@ private function processInTrackBlogComment() {
 
 	private function processInTrackLove() {
 		//BADGE_LOVE
-		if ( empty( $this->mCounters[BADGE_LOVE] ) ) {
-			$this->mCounters[BADGE_LOVE][COUNTERS_COUNTER] = 1;
+		if( empty( $this->mCounters[ BADGE_LOVE ] ) ) {
+			$this->mCounters[ BADGE_LOVE ][ COUNTERS_COUNTER ] = 1;
 		} else {
-			if ( $this->mCounters[BADGE_LOVE][COUNTERS_DATE] == date( 'Y-m-d' ) ) {
+			if( $this->mCounters[ BADGE_LOVE ][ COUNTERS_DATE ] == date( 'Y-m-d' ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Y-m-d" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)